apprunner-cdk-dotnet/README.md

```diff
@@ -0,0 +1,59 @@
+# AWS AppRunner: Deploying Web APIs with CDK
+
+The CDK stack that creates an AWS App Runner service to deploy a .NET web API application.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/apprunner-cdk-dotnet
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [.NET 6](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) installed
+* [AWS Cloud Development Kit](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) (AWS CDK) installed
+
+## Deployment Instructions
```

Comment:
Could you kindly mention explicitly that the following steps are for a Windows machine?

Reply:
Mentioned instruction for windows.